packages/manager/src/views/HomePage/components/WebpropertiesTable/VideoGuidesCard.tsx

```diff
@@ -0,0 +1,51 @@
+import { pageLinks } from '@app/links';
+import { Card, CardBody, CardHeader, Grid, GridItem, SplitItem } from '@patternfly/react-core';
+
+export const VideoGuidesCard = (): JSX.Element => {
+  const videos = [
+    {
+      title: 'SPAship Manager and CLI Guide',
+      link: 'https://drive.google.com/file/d/150OyktZdmqMXKwNS1mDkqIo7ZvgyCCGp/view'
+    },
+    {
+      title: 'Containerized deployments complete workflow',
+      link: 'https://drive.google.com/file/d/1g-obdU_RT1kQOgDKM150s_eVyiYUElKa/view?usp=sharing'
+    },
+    {
+      title: 'SPAship CLI Instruction',
+      link: 'https://drive.google.com/file/d/1G4B8AKu_8M6fVEMd8WtXHgLhf__Xl2mt/view?usp=sharing'
+    }
+  ];
```

Comment:
read these from the env variables or use the `documentation` api

Reply:
Done,